.gitignore

```
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+.yarn/install-state.gz
```

```suggestion
.yarn/*
!.yarn/cache
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```

yarn berry 공식 문서에서 pnp 기능을 사용하는 것을 권장하네요~
yarn install 시 node_modules이 생성되면 yarn berry 사용하는 이유가 없으니 저희도 적용해볼까요~~?!??!? CI 단이나 배포 파이프라인 등에서 반복되는 install 시간을 단축 시켜서 CI 실행시간 및 배포 시간이 대폭 감소된다고 하네요~~!!

공식 문서: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
zero-install 설명 문서: https://helloinyong.tistory.com/341
적용 예시: https://blog.dramancompany.com/2023/02/%EB%A6%AC%EB%A9%A4%EB%B2%84-%EC%9B%B9-%EC%84%9C%EB%B9%84%EC%8A%A4-%EC%A2%8C%EC%B6%A9%EC%9A%B0%EB%8F%8C-yarn-berry-%EB%8F%84%EC%9E%85%EA%B8%B0/

---

캬.. 너무 좋은 레퍼런스 감사합니다 👍👍

생각해보니 zero install 관련 설정을 하나도 안해놨네요😅